pelican/__init__.py

```diff
@@ -466,14 +466,22 @@ def listen(server, port, output, excqueue=None):
             excqueue.put(traceback.format_exception_only(type(e), e)[-1])
         return
 
-    logging.info("Serving at port %s, server %s.", port, server)
+    serve_msg = "Serving site at: {}:{}".format(server, port)
+    logging.info(serve_msg)
     try:
+        print("{} - Tap CTRL-C to stop".format(serve_msg))
         httpd.serve_forever()
     except Exception as e:
         if excqueue is not None:
             excqueue.put(traceback.format_exception_only(type(e), e)[-1])
         return
 
+    except KeyboardInterrupt:
+        stop_msg = "Keyboard interrupt received. Shutting down server."
+        logger.info(stop_msg)
+        print("\n{}".format(stop_msg))
```

Comment:
Shouldn't we only have one of either `logger.foo` or `print`, otherwise it'll double-output the message (assuming the logging visibility is high enough)?